### DISCUSSION TITLE
Replace string in long text field

### DISCUSSION THREAD

**Discussion by fisherbt** [rwnx/pynonymizer#132] [General]
Is it possible to replace a string within a long text field? I'm looking to replace email addresses within a column that contains the contents of email messages. Thanks!

**Comment by fisherbt** [comment]
Handled with a REGEXP_REPLACE sql statement in an "after" script.

**Reply by rwnx** [reply]
Hi, that sounds like a good fit. the other suggestion that might be of use to you is the [literal](https://github.com/rwnx/pynonymizer/blob/master/doc/strategyfiles.md#column-strategy-literal) field type, but I haven't looked further into this.
